### PR TITLE
feat(frontend): implement employer evaluation

### PR DESCRIPTION
**Pull request by thomas-mauran**
image

- fix on messaging app bar
- evaluation of an employer

### CODE REVIEW

File: front/src/components/utils/StarRatingSelector.tsx
Comment (Kuruyia) [inline front/src/components/utils/StarRatingSelector.tsx:1]: Maybe replace our `StarRating` component with this component so we don't have two similar things. We can pass a `readonly` prop to know if the rating should be able to be updated.

File: front/src/pages/employer/EmployerEvaluationPage.tsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+ */
+const EmployerEvaluationPage: FC<EmployerEvaluationPageProps> = ({
+  route,
+  navigation,
+}) => {
+  const { id: employerId } = route.params;
+
+  // API calls
+  const { data: employer, refetch: refetchEmployer } =
+    useGetEmployerQuery(employerId); // Replace '123' with the actual employer ID
```
Comment (Kuruyia) [inline front/src/pages/employer/EmployerEvaluationPage.tsx:60]:
```suggestion
    useGetEmployerQuery(employerId);
```

**Review by Kuruyia — CHANGES_REQUESTED** [review]

File: front/src/components/utils/StarRatingSelector.tsx
Comment (thomas-mauran) [reply front/src/components/utils/StarRatingSelector.tsx:1]: done

File: front/src/pages/employer/EmployerEvaluationPage.tsx (hunk 1 above)
Comment (thomas-mauran) [reply front/src/pages/employer/EmployerEvaluationPage.tsx]: done

**Review by Kuruyia — APPROVED** [review]

**Pull request merged**